lib/insights.js

```diff
@@ -0,0 +1,55 @@
+//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+'use strict';
+
+const debug = require('debug')('appinsights');
+
+function createWrappedClient(propertiesToInsert, client) {
+  const c = {};
+  if (client) {
+    c.trackEvent = addProperties.bind(client, client.trackEvent, propertiesToInsert, 1, true);
```

Comment:
Not quite understand this '1' here. Is it a common sense position for trackeEvent property?